app/src/main/java/hearsilent/zeplin/activity/ProjectActivity.kt

```diff
@@ -45,24 +50,22 @@ class ProjectActivity : AppCompatActivity(), View.OnClickListener {
 
         button_empty.setOnClickListener(this)
 
+        recyclerView.adapter = mScreenAdapter
         fetchScreens()
     }
 
     private fun fetchScreens() {
         textView_empty.visibility = View.GONE
         button_empty.visibility = View.GONE
         swipeRefreshLayout.isRefreshing = true
-        NetworkHelper.getScreens(this, mProjectModel!!.id, object : ScreensCallback() {
+        val dataSourceFactory = ScreenModelDataSourceFactory(this, mProjectModel!!.id, object : ScreensCallback() {
             override fun onSuccess(screens: List<ScreenModel>) {
                 if (isFinishing) {
                     return
                 }
                 runOnUiThread {
                     swipeRefreshLayout.isRefreshing = false
                     recyclerView.visibility = View.VISIBLE
-                    recyclerView.setHasFixedSize(true)
```

Comment:
Why don't need `setHasFixedSize` anymore?

Reply:
In my case, sometimes the screen picture size is not fixed, but I think set setHasFixedSize is true that is ok.